.gitignore

```diff
@@ -4,5 +4,6 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. However, ensure that this entry is necessary for your project setup. If your virtual environment is named differently, you might need to adjust this entry accordingly.